turbine/src/cluster_nodes.rs

```diff
@@ -409,6 +415,11 @@ fn dedup_tvu_addrs(nodes: &mut Vec<Node>) {
             // deterministic shuffle.
             return node_stake > 0u64;
         };
+        // Do not delete our own identity under any circumstances
+        // https://github.com/anza-xyz/agave/issues/5356
```

Comment:
I'm generally against including the issue as a comment (our codebase would be half github links), but if others are in favor, I'll play along